src/ophys_etl/transforms/convert_rois.py

```diff
@@ -0,0 +1,193 @@
+from pathlib import Path
+
+from argschema import ArgSchema, ArgSchemaParser
+from argschema.fields import (List, InputFile, Str, OutputFile, Float,
+                              Nested, Int, Bool)
+import marshmallow as mm
+import numpy as np
+import h5py
+
+from ophys_etl.transforms.roi_transforms import (binarize_roi_mask,
+                                                 suite2p_rois_to_coo,
+                                                 coo_rois_to_old)
+from ophys_etl.transforms.data_loaders import get_max_correction_border
+
+
+class BinarizeAndCreationException(Exception):
+    pass
+
+
+class BinarizeAndCreateROIsInputSchema(ArgSchema):
+
+    suite2p_stat_path = Str(
+        required=True,
+        description=("Path to s2p output stat file containing ROIs generated "
+                     "during source extraction")
+    )
+
+    motion_corrected_video = Str(
+        required=True,
+        description=("Path to motion corrected video file *.h5")
+    )
+
+    motion_correction_values = InputFile(
+        required=True,
+        description=("Path to motion correction values for each frame "
+                     "stored in .csv format")
+    )
+
+    output_json = OutputFile(
+        required=True,
+        description=("Path to a file to write output data.")
+    )
+
+    maximum_motion_shift = Float(
+        default=30.0,
+        required=False,
+        allow_none=False,
+        description=("The maximum allowable motion shift for a frame in pixels"
+                     " before it is considered an anomaly and thrown out of "
+                     "processing")
+    )
+
+    abs_threshold = Float(
+        default=None,
+        required=False,
+        allow_none=True,
+        description=("The absolute threshold to binarize ROI masks against. "
+                     "If not provided will use quantile to generate "
+                     "threshold.")
+    )
+
+    binary_quantile = Float(
+        default=None,
```

Comment:
Since the description states that the default is `0.1` wouldn't it be better to set the default to 0.1 and remove the `allow_none=True`?

Reply:
Removed the None type and have it defaulted to 0.1